.github/workflows/release.yml

```
@@ -68,8 +86,8 @@ jobs:
           path: binaries
           merge-multiple: true
 
-      - name: Create release
+      - name: Upload assets to release
         uses: softprops/action-gh-release@v1
```

actionlint (1.7.12)

[error] the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)